fastembed/late_interaction/colbert.py

```diff
@@ -189,7 +206,30 @@ def load_onnx_model(self) -> None:
             cuda=self.cuda,
             device_id=self.device_id,
         )
-        self.mask_token_id = self.special_token_to_id["[MASK]"]
+        self.mask_token_id, self.mask_token = next(
+            (
+                (self.special_token_to_id[token], token)
+                for token in self.MASK_TOKENS
+                if token in self.special_token_to_id
+            ),
+            (None, None),
+        )
```

Comment:
this construction is hard to read, it might be better to unwrap it and right explicitly with the regular `if` conditions